tokio-macros/src/entry.rs

```diff
@@ -513,6 +531,9 @@ impl ItemFn {
             attr.to_tokens(&mut tokens);
         }
 
+        // Add generated macros at the end, so macros processed later are aware of them.
+        generated_attrs.to_tokens(&mut tokens);
```

Comment:
You don't actually add any tests that would catch if this was added at the start rather than the end. Would there be an easy test we can add that would test that?

Reply:
Test refined to below.

```
#[tokio::test]
#[tokio::test]
async fn test_has_generated_second_test_attr() {}
```

It should error with "the `async` keyword is missing from the function declaration" before this pr as we prepend test attribute before.

Reply:
Now, the later `#[tokio::test]` complains, as it detect test attribute generated by one before it.

Reply:
Using `#[tokio::test]` twice is clever! Nice!